### PR TITLE
ref: codecov badge & tox in requirements dev

### PR DESCRIPTION
**Pull request by armenzg**

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -5,3 +5,4 @@ pytest>=7.1.1
 pytest-cov>=3.0.0
 python-dotenv>=0.20.0
 responses>=0.20.0
+tox>=3.25.1
```
Comment (asottile-sentry) [inline requirements-dev.txt:8]: generally this is too late and will install tox inside tox which isn't what you want.  the usual approach for this is to have tox ~somewhat globally available

File: .envrc [hunk 2]
```diff
@@ -15,6 +15,7 @@ pre-commit install >/dev/null
 dotenv_if_exists .env
 
 if [ ! -d .venv ]; then
+    python3 -m pip install tox
     python3 -m venv .venv
     .venv/bin/pip install -U pip
     .venv/bin/pip install wheel
```
Comment (asottile-sentry) [inline .envrc:21]: hmmm oh I see now -- I usually use `tox --devenv venv` for this usecase.  I don't think this should attempt to globally install `tox` 🤔

Comment (armenzg) [reply .envrc]: It complains with:
```
  ValueError: invalid pyproject.toml config: `project.name`.
  configuration error: `project.name` must be pep508-identifier
```

Comment (asottile-sentry) [reply .envrc]: ah it wants it to be a `dashed-name` -- maybe set `name = sentry-github-actions-app` ?

**Review by asottile-sentry — APPROVED** [review]

File: .envrc (hunk 2 above)
Comment (armenzg) [reply .envrc]: Thanks! I will look into it another time.

**Pull request merged**